### PR TITLE
Feature/unit tests

### PR DESCRIPTION
**Pull request by levelfivehub**
Unit tests for PhpCode.

### CODE REVIEW

File: test/TestAsset/Dummy.php [hunk 1]
```diff
@@ -9,5 +9,8 @@
 
 namespace ZendTest\Serializer\TestAsset;
 
-class Dummy
```
Comment (Maks3w) [inline test/TestAsset/Dummy.php:12]: Revert changes in this file

**Comment by levelfivehub** [comment]
Reverted.

File: test/Adapter/PhpCodeTest.php [hunk 2]
```diff
@@ -26,102 +24,54 @@ public function setUp()
...
     {
-        $value    = '100';
-        $expected = 100;
-
-        $data = $this->adapter->unserialize($value);
-        $this->assertEquals($expected, $data);
+        $this->assertEquals($unserialized, $this->adapter->unserialize($serialized));
     }
 
-/* TODO: PHP Fatal error:  Call to undefined method stdClass::__set_state()
```
Comment (Maks3w) [inline test/Adapter/PhpCodeTest.php:115]: Restore this comment block

Comment (levelfivehub) [inline test/Adapter/PhpCodeTest.php:115]: Done

**Comment by levelfivehub** [comment]
Thanks, we should at some point look at the unserialize (@TODO).

**Pull request merged**